server/models/index.js

```diff
@@ -1,11 +1,9 @@
-'use strict'
-
 const fs = require('fs')
 const path = require('path')
 const Sequelize = require('sequelize')
 const basename = path.basename(__filename)
-const env = process.env.NODE_ENV || 'development'
-const config = require(path.join(__dirname, '/../config/config.js'))[env]
+const env = require('../config').env
+const config = require('../config/config.js')[env]
```

Comment:
We should be consistent in using either `.env` or `[env]` (either is fine).

Reply:
You can't use `config.env` in this case as `env` is a string used to select a field on the config file, i.e. `config['development']`.

I think the config file structure needs a improving in these projects in future.

Reply:
Oh of course, I misread it! Didn't look at the lack of quotes in the second one.